etc/kayobe/kolla/config/fluentd/filter/02-audit.conf

```diff
@@ -0,0 +1,12 @@
+<match openstack_python>
+    @type rewrite_tag_filter
+    capitalize_regex_backreference yes
+    rewriterule1 python_module ^oslo.messaging.notification.+$ openstack_audit
+</match>
+
+<filter openstack_audit>
```

Comment:
For now I've avoided renaming the `payload` field to `cadf` because it seems like I need to either install a new fluend plugin, or do a deep copy of the `payload` field and then remove the original field. I guess you might also be able to use custom ruby syntax. Logstash seems a lot simpler.

https://groups.google.com/forum/#!topic/fluentd/ewC1JzdZd0s